state.go

```diff
@@ -15,6 +15,21 @@ import (
 
 type NodeStateType int
 
+func (t NodeStateType) metricsString() string {
+	switch t {
+	case StateAlive:
+		return "alive"
+	case StateDead:
+		return "dead"
+	case StateSuspect:
+		return "suspect"
+	case StateLeft:
+		return "left"
+	default:
+		return ""
```

Comment:
It might make sense to return something like:
```
// Should not happen, sign that you should add a new case above.
return fmt.Sprintf("unhandled-value-%d", t)
```

Reply:
good call.  updated.